.travis.yml

```diff
@@ -15,11 +15,11 @@ install:
 - gem update --system
```

Comment:
We should upgrade ruby here too.  Could you use `2.6.5` as recommended in the [microsite docs](https://47degrees.github.io/sbt-microsites/docs/getting-started/)?

Reply:
Of course, fixed. I somehow missed the entire tut->mdoc migration